ProblemsSolution.Tests/ProblemsSolutionTest.cs

```diff
@@ -0,0 +1,32 @@
+using NUnit.Framework;
+
+namespace ProblemsSolution.Tests
+{
+    [TestFixture]
+    public class Tests
+    {
+
+        [TestCase (new int[] {1,2,3,4,5}, ExpectedResult=6, Description="Check for array with even numbers.")]
+        [TestCase (new int[] {3, 7, 0, 1}, ExpectedResult=0, Description="Check for array with even numbers and a zero.")]
+        public int SumArrayEvenNumbersTest(int[] numbers)
+        {
+            return ProblemsSolution.ArrayProblemsSolution.SumArrayEvenNumbers(numbers);
+        }
+
+        [TestCase (new int[] {12, 22, 1, 3}, ExpectedResult=new int[] {12, 22}, Description="Check for array with more than 2 elements.")]
+        [TestCase (new int[] {3}, ExpectedResult=new int[] {3}, Description="Check for single element array.")]
+        [TestCase (new int[] {7, 3}, ExpectedResult=new int[] {3, 7}, Description="Check for 2 element array.")]
+        [TestCase (new int[] {}, ExpectedResult=new int[] {}, Description="Check for empty array.")]
```

Comment:
How about test cases with negative numbers?

Reply:
> How about test cases with negative numbers?

Yeah it seems that when it comes to negative numbers the array order is reversed, will fix.

Reply:
Fixed 45ff35598dc7149787063f58f7e9d84a406b16b2.